eos-tech-support/eos-hack-reboot-to-first-contact

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+
+usage () {
+    cat <<EOF
+Usage: $0
+Reboot the machine and deletes all users in the system,
+so you can execute the first contact experience from
+scratch.
+
+  -h, --help                display this help and exit
+
+EOF
+}
+
+args=$(getopt -o h -l "help" -n "eos-hack-reboot-to-first-contact" -- "$@")
+
+if [ $? != 0 ]; then
+    exit 1
+fi
+
+eval set -- "$args"
+
+while true; do
+    case "$1" in
+        -h|--help)
+        usage
+        exit 0
+        ;;
+    --)
+        shift
+        ;;
+    *)
+        break
+        ;;
+    esac
+done
+
+# Check that script was run with superuser privileges.
+if [[ $EUID != 0 ]]; then
+    echo "$0 requires superuser privileges."
+    exit 1
+fi
+
+echo "This script will reboot the system and DELETE ALL USERS so you can"
+echo "restart the Hack first contact experience and create a new user."
```

Comment:
It may be worth noting here that this script will dump you at the Endless logo for a potentially long period of time while the script runs with no user-visible feedback.